### DISCUSSION TITLE
please update cheat on this games

### DISCUSSION THREAD

**Discussion by bangizz0027** [GoldHEN/GoldHEN_Cheat_Repository#65] [Cheat requests]
digimon world next order
CUSA05392 – EUR 1.00 to 1.02

request for add this game 
digimon world 4
SLES53405

stranger of paradise
final fantasy origin
CUSA29578 – USA
request for update to 1.12

**Comment by kmeps4** [comment]
Hello. Please go to https://discord.com/invite/bUEQj9p and join PS4Trainer Discord and do the request there.
Thx.
